.gitlab/notify/notify.yml

```diff
@@ -64,8 +64,12 @@ notify_github:
     - !reference [.except_mergequeue]
     - !reference [.except_main_or_release_branch]
     - !reference [.except_no_tests_no_deploy]
-    - !reference [.on_a7]
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Why is this removed?

Reply:
Otherwise the `.on_a7` triggers the job, I'll try to rewrite it